packages/flutter/lib/src/physics/spring_simulation.dart

```diff
@@ -37,17 +44,32 @@ class SpringDescription {
     double ratio = 1.0,
   }) : damping = ratio * 2.0 * math.sqrt(mass * stiffness);
 
-  /// The mass of the spring (m). The units are arbitrary, but all springs
-  /// within a system should use the same mass units.
+  /// The mass of the spring (m).
+  ///
+  /// The units are arbitrary, but all springs within a system should use
+  /// the same mass units.
+  ///
+  /// The greater the mass, the larger the amplitude of oscillation,
+  /// and the longer the time to return to the equilibrium position.
   final double mass;
 
-  /// The spring constant (k). The units of stiffness are M/T², where M is the
-  /// mass unit used for the value of the [mass] property, and T is the time
-  /// unit used for driving the [SpringSimulation].
+  /// The spring constant (k).
+  ///
+  /// The units of stiffness are M/T², where M is the mass unit used for the
+  /// value of the [mass] property, and T is the time unit used for driving
+  /// the [SpringSimulation].
+  ///
+  /// Stiffness defines the spring constant, which measures the strength of
+  /// the spring. A stiff spring applies more force to the object that is
+  /// attached when the spring is not at the rest position.
```

Comment:
Another physics nit. Rephrasing the last sentence here:

 > A stiff spring applies more force to the object that is attached for some deviation from the rest position.